From 8dc8f22c686b434dc6c9a5510c5e7b16fe330ecf Mon Sep 17 00:00:00 2001
From: Nicholas Strugnell <nstrug@redhat.com>
Date: Wed, 31 Jul 2013 16:24:20 +0100
Subject: [PATCH 008/117] sysprep: rhn_systemid: delete osad-auth.conf file in
 RHN configuration (cherry picked from commit
 94e5ea6b5ef0d4e4f2d30b51cb5dcce175819456)

---
 sysprep/sysprep_operation_rhn_systemid.ml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sysprep/sysprep_operation_rhn_systemid.ml b/sysprep/sysprep_operation_rhn_systemid.ml
index 99ab5e1..6ccf5af 100644
--- a/sysprep/sysprep_operation_rhn_systemid.ml
+++ b/sysprep/sysprep_operation_rhn_systemid.ml
@@ -28,6 +28,7 @@ let rhn_systemid_perform g root =
   match typ, distro with
   | "linux", "rhel" ->
     (try g#rm "/etc/sysconfig/rhn/systemid" with G.Error _ -> ());
+    (try g#rm "/etc/sysconfig/rhn/osad-auth.conf" with G.Error _ -> ());
     []
   | _ -> []
 
-- 
2.5.0

